src/main/webapp/app/shared/components/code-button/code-button.component.ts

```
@@ -347,4 +384,40 @@
+            return;
+        }
+        newWindow.name = 'Theia-IDE';
+
+        const form = document.createElement('form');
+        form.method = 'GET';
+        form.action = reactURL;
+        form.target = newWindow.name;
+
+        // Loop over data element and create input fields
+        for (const key in data) {
+            if (data.hasOwnProperty(key)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)